src/footer/footer.html

```diff
@@ -27,7 +27,7 @@
           <span>Docs</span>
         </a>
 
-        <a href-route="#">
+        <a href="https://kolektivo.typeform.com/to/TPMzQKFE">
```

Comment:
this is an external link.  please follow the design guidelines for anchor tags with external links